src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/DoNotUseInsecureDtdProcessing.cs

```diff
@@ -23,11 +23,42 @@ public sealed class DoNotUseInsecureDtdProcessingAnalyzer : DiagnosticAnalyzer
     {
         internal const string RuleId = "CA3075";
 
```

Comment:
Tagging @dotpaul for reviewing changes to this file. I think we probably want to create a separate PR targeting 2.9.x branch for all changes to security rules.